lib/comment/view/create_comment_page.dart

```diff
@@ -93,6 +94,10 @@ class _CreateCommentPageState extends State<CreateCommentPage> {
   /// Used for restoring and saving drafts
   SharedPreferences? sharedPreferences;
 
+  /// Whether to view source for posts or comments
+  bool postViewSource = false;
```

Comment:
I believe we can only show either post or comment at the same time (they're mutually exclusive since you can only reply to a post or comment at one time). We could get away with just using a single parameter `viewSource` here!

Reply:
Good point! Done!